client/src/components/chefs-info/index.tsx

```diff
@@ -20,11 +15,11 @@ const ChefsInfo: React.FC <ChefsInfoProps> = ({chefs}) => {
   return (
     <>
       {chefs.map(chef => (
-        <ChefsInfoWrapper data-aos="fade-down" key={chef.id}>
-          <ChefsInfoPhoto photo={chef.photo} />
-          <ChefsInfoName>{chef.name}</ChefsInfoName>
-          <ChefsInfoPosition>{chef.position}</ChefsInfoPosition>
-        </ChefsInfoWrapper>
+        <_.ChefsInfoWrapper data-aos="fade-down" key={chef.id}>
```

Comment:
why do you need `_.`? it's a bad practise

Reply:
i corrected it